app/main.py

```diff
@@ -1,3 +1,65 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km):
+        self.km = km
+
+    def __str__(self):
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self):
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other):
+        if not isinstance(other, Distance):
+            self.km += other
+        else:
+            self.km += other.km
+        return Distance(self.km)
+
+    def __iadd__(self, other):
+        if not isinstance(other, Distance):
+            self.km = self.km + other
+        else:
+            self.km = self.km + other.km
+        return self
+
+    def __mul__(self, other):
+        if not isinstance(other, Distance):
+            self.km *= other
+        else:
+            self.km *= other.km
+        return Distance(self.km)
+
+    def __truediv__(self, other):
+        if not isinstance(other, Distance):
+            self.km /= other
+        else:
+            self.km /= other.km
+        return Distance(round(self.km, 2))
+
+    def __lt__(self, other):
+        if not isinstance(other, Distance):
+            return self.km < other
+        return self.km < other.km
+
+    def __gt__(self, other):
+        if not isinstance(other, Distance):
+            return self.km > other
+        return self.km > other.km
+
+    def __eq__(self, other):
+        if not isinstance(other, Distance):
+            return self.km == other
+        return self.km == other.km
+
+    def __le__(self, other):
+        if not isinstance(other, Distance):
+            return self.km <= other
+        return self.km <= other.km
```

Comment:
optimize this method: use the method you already wrote. Don't forget to use it as a magic method